### DISCUSSION TITLE
trojan:Win64/lazy!mtb

### DISCUSSION THREAD

**Discussion by wizvrd1** [YimMenu/YimMenu#2988] [Q&A]
Hello, I have tried to download yimmenu and when it is downloaded I receive messages from Windows Defender about viruses that are in the folder, has anyone experienced this?

**Comment by Yimura** [comment]
I'm gonna take the answer that UnknownCheats usually gives to these types of questions:

> Your antivirus flagging this file is very likely to be a false positive.
Gamehacks are often recognized as or confused with malware by antivirus services due to the way these programs interact with system processes, memory and the operating system itself. Gamehacks may also use some form of obfuscation to circumvent anti-cheat detection or prevent others from reverse engineering the author's work.

**Reply by wizvrd1** [reply]
But is it safe to allow yimmenu? I was using it for 2 weeks quietly, until GTA was updated and I received the Windows Defender virus ads

**Reply by Yimura** [reply]
Well, YimMenu is being used by a lot of people. It has probably since been identified by "Microsoft" as a virus which it's not. However I am a biased party so in the end it's up to you if you feel comfortable with using it. Do keep in mind a lot of other people use YimMenu, it's open source and if you still don't like using the pre-built binary then you can download the source and compile it yourself locally. However if you're going down that route please do take the time to learn it yourself as well as read through the guide on how to setup your PC as well.

https://github.com/YimMenu/YimMenu/wiki/Setup-your-PC-for-YimMenu-Development

**Reply by wizvrd1** [reply]
thank you for your help
